package.json

```diff
@@ -78,7 +78,6 @@
     "ember-cli-import-polyfill": "^0.2.0",
     "ember-cli-is-package-missing": "^1.0.0",
     "ember-cookies": "^0.0.10",
-    "ember-getowner-polyfill": "^1.0.1",
```

Comment:
I removed that because of `ember-getowner-polyfill is not required for Ember 2.3.0 and later, please remove from your `package.json`` at build time, but I think if you want to keep ember-simple-auth compatible with ember 1.12+ this dependency is required, right ?